match/README.md

```diff
@@ -358,6 +358,19 @@ If you want to manually decrypt a file you can.
 openssl aes-256-cbc -k "<password>" -in "<fileYouWantToDecryptPath>" -out "<decryptedFilePath>" -a -d
 ```
 
+### <a name="import_certificate"></a>Import an existing certificate
+
+To quickly and easily get started with `match`, it is adviced to revoke your existing certificates. Although, if revoking code signing certificates is not an available option for you, match provides an `import` feature that let's you import an existing code signing identity certificate, to your match's repository.
+
+Before using this feature it is required that the code signing certificate you want to import is properly installed in your Keychain and it is associated with an active code signing identity registered in your Apple Developer account. Check this [fastlane.docs page](<to-do-link-to-fastlane-docs-page>) to easily speed up with this requirements.
+
+To import your code signing certificate, P12 encoded, run:
+
+```
+match import "<path-to-p12-code-signing-certificate>" "<p12-password-if-not-empty-string>"
+```
+
+The command will validate the certificate that you want to import and associate it with its Apple Developer's representation. The information retrieved from the Apple Developer account will make sure that the certificate to import is properly processed and analyzed by `match` that finally imports, and encrypts it, in its repository.
```

Comment:
"The command will validate the certificate and associate it with the Apple Developer Portal entity. This will ensure that the certificate is correctly processed by `match` and imported into the repository."?